sympy/core/relational.py

```diff
@@ -196,8 +196,17 @@ def negated(self):
         #      b, evaluate=evaluate)))(*self.args, evaluate=False)
         return Relational.__new__(ops.get(self.func), *self.args)
 
-    def _eval_evalf(self, prec):
-        return self.func(*[s._evalf(prec) for s in self.args])
+    def _eval_evalf_options(self, n, options):
+        #from sympy.core.evalf import evalf
+        subs, quad = None, None
+        if 'subs' in options:
+            subs = options['subs']
+        if 'quad' in options:
+            quad = options['quad']
+        #return self.func(*[evalf(s, prec, options) for s in self.args])
+        return self.func(*[s.evalf(n, subs=subs, maxn=options['maxn'], chop=options['chop'],
+                            strict=options['strict'], quad=quad,
+                            verbose=options['verbose']) for s in self.args])
```

Comment:
This is a horrible way to recurse. We should make it so that `_eval_evalf_options` takes the same arguments as the `evalf` function and then recurse using the `evalf` function.

Reply:
Yes, agreed.